src/steamship/agents/schema/memory_strategies.py

```diff
@@ -0,0 +1,73 @@
+from typing import List
+
+import tiktoken
+
+from steamship import Block
+from steamship.agents.memory.chathistory import ChatHistory, MemoryStrategy
+from steamship.data.tags.tag_constants import RoleTag
+
+
+class NoMemory(MemoryStrategy):
+    def messages(self, chat_history: ChatHistory) -> List[Block]:
+        return []
+
+
+def is_user_message(block: Block) -> bool:
```

Comment:
Can this be part of Block? That helps with discoverability. 

Sidenote: These type of functions make me want to create a `ChatMessage` wrapper again.

Reply:
`ChatMessage` definitely has some appeal. Let's either go all in on `Block` or subclass/wrap and add `ChatMessage`. I feel like I'm constantly torn on which one I like :).

Reply:
I'm team ChatMessage after I've found myself adding it back in for GGPT. Sending Blocks to people didn't feel natural, ChatMessages did.